dashboard/app/models/user.rb

```diff
@@ -2197,6 +2197,11 @@ def record_soft_delete
     )
   end
 
+  def has_pilot_experiment?(pilot_name)
+    return false unless pilot_name
+    SingleUserExperiment.enabled?(user: self, experiment_name: pilot_name)
```

Comment:
From what I understand, it's a table that records each instance of a user-pilot relationship (instead of the Experiments table, which just rattles off the list without having an entry for each person enrolled).